### DISCUSSION TITLE
How to disable abbrevs/dictionary expansion?

### DISCUSSION THREAD

**Discussion by azzamsa** [abougouffa/minemacs#85] [Q&A] [answered]
Hi.

I have set `evil-want-abbrev-expand-on-insert-exit` to `ni`, but I can't prevent abbrevs/dictionary to expand.

![Kooha-2023-06-21-18-10-18](https://github.com/abougouffa/minemacs/assets/17734314/c1285342-45b9-427d-9e3e-334affad7d32)

Every time I wan to write a function named `today()` it expands at unfortunate times.

Thanks! :heart:

**Comment by abougouffa** [comment]
I have no idea! Never seen such a behavior!

**Reply by azzamsa** [reply]
Any suggestion on how to debug it?

**Reply by abougouffa** [reply, marked as answer]
Try to disable `tempel-abbrev-mode` in the current buffer!
